entries/z/zaius.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Zaius": {
+    "domain": "zaius.com",
+    "additional-domains": [
+      "app.zaius.com"
+    ],
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.zaius.com/hc/en-us/articles/360039477354",
+    "notes": "Only User Admins can enable two-factor authentication",
+    "keywords": [
+      "social"
```

Comment:
I'm not sure that the `social` category fits very well here. Maybe `communication` or `other` might be better.
It looks similar to "Listrak" which is in the `other` categpry.

Reply:
category changed  `other`